docs/HowToFAQ/FAQNestedModule.md

```diff
@@ -0,0 +1,55 @@
+---
+title: Can I access the members of an outer module from its inner module?
+---
+
+## Question
+
+Can I access the members of an outer module from its inner module?
+```dafny
+{% include_relative FAQNestedModule.dfy %}
+```
+
+## Answer
+
+No. The above example gives the error messages
+```text
+{% include_relative FAQNestedModule.txt %}
```

Comment:
Just using 'dafny /useBaseNameForFileName' . Many examples, like this one, are in included files so that they can be part of the test suite, which will then alert to any changes in dafny that would make the FAQ obsolete.